security/crowdsec/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		crowdsec
-PLUGIN_VERSION=		1.0.8
+PLUGIN_VERSION=		1.0.9
```

Comment:
I'd prefer to squash this into 1.0.8 revision 1 as a hotfix (PLUGIN_REVISION) since it's not a feature and we will issue this before 24.1.8 is out most likely.

Reply:
ok thanks